test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -59,6 +59,33 @@ export function setup(opts: minimist.ParsedArgs) {
 			await app.workbench.sqlNotebook.waitForColorization('6', 'mtk1'); // employees
 		});
 
+		it('can enter and exit edit mode and navigate using keyboard nav', async function () {
+			const app = this.app as Application;
+			await app.workbench.sqlNotebook.newUntitledNotebook();
+			await app.workbench.sqlNotebook.addCellFromPlaceholder('Code');
+			await app.workbench.sqlNotebook.waitForPlaceholderGone();
+			assert(await app.workbench.sqlNotebook.isCodeCellInEditMode(), 'code cell should be in edit mode');
+			await app.code.dispatchKeybinding('escape');
+			assert(!(await app.workbench.sqlNotebook.isCodeCellInEditMode()), 'code cell should not be in edit mode');
```

Comment:
For all these make sure you indicate where in the process it failed, so like on line 82 this should end with something like `after hitting escape`